src/steam.c

```diff
@@ -165,18 +165,6 @@ void pressure_vessel(int argc, const char** argv, int nextarg, const char* prog)
             newargv[i-nextarg] = argv[i];
         }
     }
-    //setenv("BOX64_PREFER_EMULATED", "1", 1);
-    //setenv("BOX86_PREFER_EMULATED", "1", 1);
-
-//setenv("BOX64_TRACE_FILE", "/home/seb/trace64-%pid.txt", 1);
```

Comment:
Please keep that in place, this is for my personal use...

Reply:
recovered. sorry I thought these were some leftover.